src/test/java/uk/gov/mca/beacons/service/beacons/BeaconsControllerIntegrationTest.java

```diff
@@ -95,6 +107,47 @@ void requestAllBeaconControllerShouldReturnSomeBeacons() {
       .exists();
   }
 
+  @Test
+  void requestBeaconControllerShouldReturnBeaconByUuid() {
```

Comment:
It would be good if we could add an additional integration test around handling of no emergency contacts. I know it's not possible through the existing form but for the migration of data we're going to have scenarios where there are no emergency contacts.

The test should check that we can handle gracefully the fact that a beacon has no emergency contacts.

You should be able to extract the code within the `@BeforeEach` to a helper function, something like: `createRegistration(boolean withEmergencyContacts)` and default to creating a registration with an emergency contact and then for the test of checking when there are no emergency contacts set this to false